greeneeds/users/views.py

```diff
@@ -45,4 +50,49 @@ def get(self, request):
             'nickname' : user.nickname,
             'email'    : user.email
         }
-        return JsonResponse({'result': result}, status=200)
\ No newline at end of file
+        return JsonResponse({'result': result}, status=200)
+class LikeView(View):  
+    @login_decorator
+    def post(self, request):
+        try:
+            data       = json.loads(request.body)
+            user       = request.user
+            project_id = data['project_id']
+            project    = Project.objects.get(id=project_id)
+
+            if Like.objects.filter(user=user, project=project).exists():
+                Like.objects.filter(user=user, project=project).delete()
+                like_count = Like.objects.filter(project=project).count()
+                return JsonResponse({'MESSAGE': 'SUCCESS', 'LIKE_COUNT':like_count}, status=201)
```

Comment:
1. key 값은 소문자로 해주세요.
2. 상태코드 201은 생성 할 때 입니다. 보통 delete는 204를 사용하는데, 204를 사용하면 응답의 body가 존재 하지 않게 됩니다. 따라서 이 경우에는 200을 사용하면 좋을 것 같습니다.
```suggestion
                return JsonResponse({'message': 'SUCCESS', 'like_count':like_count}, status=200)
```